webpack.config.dist.js

```diff
@@ -33,6 +33,10 @@ module.exports = {
       {
         test: /\.js$/,
         loaders: ["babel"],
+        exclude: [
+          path.join(__dirname, "node_modules", "core-js"),
+          path.join(__dirname, "node_modules", "babel-runtime"),
```

Comment:
I think it might work without this

Reply:
(eg delete line 38, you only need line 37)

Reply:
When I removed the `babel-runtime` element of the list, I got the same results as before (described in the PR notes as `29` including `30` including `31` including 32` including `29`).

Reply:
Oh ok, that's odd. LGTM otherwise